GVFS/GVFS.FunctionalTests/Program.cs

```diff
@@ -84,6 +88,15 @@ public static void Main(string[] args)
                 includeCategories.Remove(Categories.ExtraCoverage);
             }
 
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                excludeCategories.Add(Categories.WindowsOnly);
+            }
+            else
+            {
+                excludeCategories.Add(Categories.LinuxOnly);
+            }
+

```

Comment:
* We already exclude `MacOnly` because of that same following block, which adds `MacOnly` to the exclusion list if the platform is not `OSPlatform.OSX`.
* I'm unsure on what you're asking with the second question here. Incorporate what new exclusions where?

Reply:
> * I'm unsure on what you're asking with the second question here. Incorporate what new exclusions where?

(To be clear, not trying to be obtuse! Just not parsing the question well.)

Reply:
My reading of @wilbaker's suggestion here is that because many of these functional tests have not yet been ported to the Mac (and are therefore Windows-only not by design, but only because of lack of time), we will want to, for now, also exclude these on Linux.  I'll make that change shortly!

Reply:
I think I'm going to leave this as-is for now, because we haven't yet run the functional tests on Linux (see github/VFSForGit#16) and so we may find that we need a different, likely larger, set of `LinuxTODO` tests.  Until we investigate, it's a little hard to know which ones from the `MacTODO` category will really be `POSIX-TODO` and apply to Linux as well, and which are Mac-specific.

But as I'm 100% certain we'll be revisiting these tests and categoroes very soon, I think I'll go ahead and merge with this as it stands, as a placeholder (projecting filesystem pun intended! :-)

Reply:
@kivikakk @chrisd8088 apologies for the confusion.

The idea in the second point was to combine the new if/else with the existing one below so that it would look something like:

```
if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
{
    excludeCategories.Add(... stuff excluded on Mac ...);
}
else if (RuntimeInformation.IsOSPlatform(Linux))
{
    excludeCategories.Add(... stuff excluded on Linux ...);
}
else
{
   excludeCategories.Add(... stuff excluded on Windows ...);
}
```

It can get tricky to keep track of what we're excluding or including, and it might be easier to keep track if we set them all in one spot.

Reply:
To clarify, when I wrote `excludeCategories.Add(... stuff excluded on Linux ...);` I meant call  `excludeCategories.Add` as many times as needed in each block to exclude what should be excluded on each platform

Reply:
Ah, makes perfect sense. I'll do a PR for this -- thanks!